### ISSUE TITLE
API "validateNames: unknown name tree name: Type"

### ISSUE THREAD

**Issue by vinser** [pdfcpu/pdfcpu#498]
pdfcpu: v0.3.13 dev
Ubuntu 20.04.4 LTS

I'm trying to use pdfcpu api in my server app and write this simple test main.go
```go
package main

import (
	"log"
	"os"

	"github.com/pdfcpu/pdfcpu/pkg/api"
)

func main() {

	out, err := api.InfoFile(os.Args[1], nil, nil)
	log.Println("-----------------")
	if err != nil {
		log.Println(err)
	} else {
		for _, s := range out {
			log.Println(s)
		}
	}
	log.Println("-----------------")
}
```
When I run it for great progit.pdf I get
```
go run main.go progit.pdf 
2022/07/16 18:21:36 -----------------
2022/07/16 18:21:36 validateNames: unknown name tree name: Type

2022/07/16 18:21:36 -----------------
```
pdfcpu info command give me good result with the same pdf file
```
pdfcpu info progit.pdf 
         PDF version: 1.4
          Page count: 538
           Page size: 595.28 x 841.89 points
............................................
               Title: Pro Git
              Author: Scott Chacon, Ben Straub
             Subject: 
        PDF Producer: Scott Chacon, Ben Straub
     Content creator: Asciidoctor PDF 2.0.8, based on Prawn 2.4.0
       Creation date: D:20220614181125+00'00'
   Modification date: D:20220614180958+00'00'
............................................
              Tagged: No
              Hybrid: No
          Linearized: No
  Using XRef streams: No
Using object streams: No
         Watermarked: No
          Thumbnails: No
            Acroform: No
............................................
           Encrypted: No
         Permissions: Full access
```  
With other pdf my main.go works well
```
go run main.go alice-adventures-in-wonderland.pdf 
2022/07/16 18:18:25 -----------------
2022/07/16 18:18:25          PDF version: 1.4
2022/07/16 18:18:25           Page count: 60
2022/07/16 18:18:25            Page size: 612.00 x 792.00 points
2022/07/16 18:18:25 ............................................
2022/07/16 18:18:25                Title: ALICE ADVENTURES IN WONDERLAND
2022/07/16 18:18:25               Author: Lewis Carroll
2022/07/16 18:18:25              Subject: 
2022/07/16 18:18:25         PDF Producer: calibre 3.40.1 [https://calibre-ebook.com]
2022/07/16 18:18:25      Content creator: calibre 3.40.1 [https://calibre-ebook.com]
2022/07/16 18:18:25        Creation date: D:20201219024004+00'00'
2022/07/16 18:18:25    Modification date: 
2022/07/16 18:18:25 ............................................
2022/07/16 18:18:25               Tagged: No
2022/07/16 18:18:25               Hybrid: No
2022/07/16 18:18:25           Linearized: No
2022/07/16 18:18:25   Using XRef streams: No
2022/07/16 18:18:25 Using object streams: No
2022/07/16 18:18:25          Watermarked: No
2022/07/16 18:18:25           Thumbnails: No
2022/07/16 18:18:25             Acroform: No
2022/07/16 18:18:25 ............................................
2022/07/16 18:18:25            Encrypted: No
2022/07/16 18:18:25          Permissions: Full access
2022/07/16 18:18:25 -----------------
```
 
Thanks

**Comment by vinser** [comment]
You can get progit.pdf here https://github.com/progit/progit2/releases/download/2.1.347/progit.pdf

**Comment by hhrutter** [comment]
Hello!

All you need to provide is a configuration with `validationMode: ValidationRelaxed`.

also on the cli check : `pdfcpu conf`

**Comment by vinser** [comment]
Hello!
Sorry, but setting, that I've made 
```go
func main() {
       // Add thees two lines
	conf := pdfcpu.NewDefaultConfiguration()
	conf.ValidationMode = pdfcpu.ValidationRelaxed

	out, err := api.InfoFile(os.Args[1], nil, conf)
...
```
does not help, result is the same `validateNames: unknown name tree name: Type`

api package is not enough for integration usage :(

**Comment by hhrutter** [comment]
Have you checked your running the latest commit?

In api_test.go your file does not show any issue when used in func TestInfo(t *testing.T)


------

### Thank you for submitting a possible bug!

Please ensure the following:

* **Your issue is based on the latest commit**
* State your OS and OS version
* When reporting a problem with a specific PDF input file please avoid stating the organization responsible for the PDFWriter - just refer to the *PDFWriter*

**Comment by vinser** [comment]
Hello!
Yes, I tried this pdf with TestInfo func result is Ok. But with my main.go I get the same error:
```
2022/07/24 07:51:23 -----------------
2022/07/24 07:51:23 validateNames: unknown name tree name: Type

2022/07/24 07:51:23 -----------------
```

Here is more detailed info you asked for:
lsb_release -d
```
Description:	Ubuntu 22.04 LTS
```
main.go
```go
package main

import (
	"log"
	"os"

	"github.com/pdfcpu/pdfcpu/pkg/api"
	"github.com/pdfcpu/pdfcpu/pkg/pdfcpu"
)

// main
func main() {
	conf := pdfcpu.NewDefaultConfiguration()
	conf.ValidationMode = pdfcpu.ValidationRelaxed

	out, err := api.InfoFile(os.Args[1], nil, conf)
	log.Println("-----------------")
	if err != nil {
		log.Println(err)
	} else {
		for _, s := range out {
			log.Println(s)
		}
	}
	log.Println("-----------------")
}
```
go.mod
```
module pdfinfo

go 1.18

require github.com/pdfcpu/pdfcpu v0.3.13

require (
	github.com/hhrutter/lzw v0.0.0-20190829144645-6f07a24e8650 // indirect
	github.com/hhrutter/tiff v0.0.0-20190829141212-736cae8d0bc7 // indirect
	github.com/pkg/errors v0.9.1 // indirect
	golang.org/x/image v0.0.0-20210220032944-ac19c3e999fb // indirect
	golang.org/x/text v0.3.6 // indirect
	gopkg.in/yaml.v2 v2.4.0 // indirect
)
```
go.sum
```
github.com/hhrutter/lzw v0.0.0-20190827003112-58b82c5a41cc/go.mod h1:yJBvOcu1wLQ9q9XZmfiPfur+3dQJuIhYQsMGLYcItZk=
github.com/hhrutter/lzw v0.0.0-20190829144645-6f07a24e8650 h1:1yY/RQWNSBjJe2GDCIYoLmpWVidrooriUr4QS/zaATQ=
github.com/hhrutter/lzw v0.0.0-20190829144645-6f07a24e8650/go.mod h1:yJBvOcu1wLQ9q9XZmfiPfur+3dQJuIhYQsMGLYcItZk=
github.com/hhrutter/tiff v0.0.0-20190829141212-736cae8d0bc7 h1:o1wMw7uTNyA58IlEdDpxIrtFHTgnvYzA8sCQz8luv94=
github.com/hhrutter/tiff v0.0.0-20190829141212-736cae8d0bc7/go.mod h1:WkUxfS2JUu3qPo6tRld7ISb8HiC0gVSU91kooBMDVok=
github.com/pdfcpu/pdfcpu v0.3.13 h1:VFon2Yo1PJt+sA57vPAeXWGLSZ7Ux3Jl4h02M0+s3dg=
github.com/pdfcpu/pdfcpu v0.3.13/go.mod h1:UJc5xsXg0fpmjp1zOPdyYcAQArc/Zf3V0nv5URe+9fg=
github.com/pkg/errors v0.9.1 h1:FEBLx1zS214owpjy7qsBeixbURkuhQAwrK5UwLGTwt4=
github.com/pkg/errors v0.9.1/go.mod h1:bwawxfHBFNV+L2hUp1rHADufV3IMtnDRdf1r5NINEl0=
golang.org/x/image v0.0.0-20190823064033-3a9bac650e44/go.mod h1:FeLwcggjj3mMvU+oOTbSwawSJRM1uh48EjtB4UJZlP0=
golang.org/x/image v0.0.0-20210220032944-ac19c3e999fb h1:fqpd0EBDzlHRCjiphRR5Zo/RSWWQlWv34418dnEixWk=
golang.org/x/image v0.0.0-20210220032944-ac19c3e999fb/go.mod h1:FeLwcggjj3mMvU+oOTbSwawSJRM1uh48EjtB4UJZlP0=
golang.org/x/text v0.3.0/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
golang.org/x/text v0.3.6 h1:aRYxNxv6iGQlyVaZmk6ZgYEDa+Jg18DxebPSrd6bg1M=
golang.org/x/text v0.3.6/go.mod h1:5Zoc/QRtKVWzQhOtBMvqHzDpF6irO9z98xDceosuGiQ=
golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e/go.mod h1:n7NCudcB/nEzxVGmLbDWY5pfWTLqBcC2KZ6jyYvM4mQ=
gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405 h1:yhCVgyC4o1eVCa2tZl7eS0r+SDo693bJlVdllGtEeKM=
gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod h1:Co6ibVJAznAaIkqp8huTwlJQCZ016jof/cbN4VW5Yz0=
gopkg.in/yaml.v2 v2.4.0 h1:D8xgwECY7CYvx+Y2n4sBz93Jn9JRvxdiyyo8CTfuKaY=
gopkg.in/yaml.v2 v2.4.0/go.mod h1:RDklbk79AGWmwhnvt/jBztapEOGDOx6ZbXqjP6csGnQ=
```

**Comment by vinser** [comment]
I found latest commit of closed Issue #485 was not applied on v0.3.13
sorry
